viite-UI/src/view/LinkPropertyLayerStyles.js

```diff
@@ -1,23 +1,23 @@
 (function(root) {
   root.LinkPropertyLayerStyles = function(roadLayer) {
-    /*
-    / Todo: Handle unknown, stroke black with question mark
-    /
+
+    // Todo: Handle unknown, stroke black with question mark
+
     var unknownRoadClassDefaultRules = [
-      new OpenLayersRule().where('roadClass').is('99').use({ strokeColor: '#000000', strokeOpacity: 1.0, externalGraphic: 'images/speed-limits/unknown.svg', pointRadius: 14})
+      new OpenLayersRule().where('type').is('roadAddressAnomaly').use({ strokeColor: '#000000', strokeOpacity: 0.8, externalGraphic: 'viite-UI/images/speed-limits/unknown.svg', pointRadius: 14})
```

Comment:
I belive we could have the same image on a different folder instead of using speedlimit.